nupic/algorithms/CLAClassifier.py

```diff
@@ -213,6 +213,31 @@ def __setstate__(self, state):
     self._version = BitHistory.__VERSION__
 
 
+  def write(self, proto):
+    proto.id = self._id
+
+    i = 0
+    statsProto = proto.init("stats", len(self._stats))
+    for (bucketIdx, dc) in enumerate(self._stats):
```

Comment:
I'd prefer at least this occurrence use `dutyCycle`. If you want to fix existing occurrences in this file that is fine as well, although might be easier to get this PR merged first and send one just for fixing existing occurrences.